### ISSUE TITLE
PicoBoard projects fail to load when using reporter blocks

### ISSUE THREAD

**Issue by ktbee** [LLK/scratch-vm#1682]
### Expected Behavior

PicoBoard projects aren't supported in Scratch 3.0, so its blocks shouldn't load.

### Actual Behavior

Project fails to load entirely, just get error that "Oops! Something went wrong" on the screen, and this error in the console:

```
raven.js:55 Error: Failed to load project from server [id=#116599254]: TypeError: Cannot set property 'parent' of null
```

### Steps to Reproduce

1) Find a project that uses PicoBoard reporter blocks in non-PicoBoard blocks (example: 116599254)
2) Try to load project, observe it fails to load (https://llk.github.io/scratch-gui/develop/#116599254)
3) Remove reporter blocks from non-PicoBoard blocks (example: 254072184)
4) Observe it loads successfully and doesn't display PicoBoard blocks (https://llk.github.io/scratch-gui/develop/#254072184)

### Operating System and Browser

Mac OS 10.13.4 Chrome 69.0.3497.100

**Comment by ktbee** [comment]
This bug is potentially similar to https://github.com/LLK/scratch-vm/pull/1524

**Comment by ktbee** [comment]
closed via #1716 !
